### ISSUE TITLE
'patch-package' is not recognized as an internal or external command

### ISSUE THREAD

**Issue by ericcornelissen** [steelbrain/package-deps#317]
After upgrading to/installing [v7.0.0](https://github.com/steelbrain/package-deps/releases/tag/v7.0.0) I'm getting the following error message:

```
$ npm install atom-package-deps

> atom-package-deps@7.0.0 postinstall C:\Users\erico\.atom\packages\linter-htmllint\node_modules\atom-package-deps
> patch-package

'patch-package' is not recognized as an internal or external command,
operable program or batch file.
[...]
```

I'm guessing this can be fixed by moving `"patch-package": "^6.2.2",` back into the `"dependencies"` section from the `"devDependencies"` section (partly reverting f16e38cdbcdeac64baa8c8433d801d89b15df1b2).

**Comment by steelbrain** [comment]
Fixed in https://github.com/steelbrain/package-deps/commit/a64ac22e7b74315221c724c3a7a2425c2c8462c2

**Comment by steelbrain** [comment]
Fix released in v7.0.1
